File: Kms.Security.Common/DataContract/EnterprisePositionDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Kms.Security.Common.Domain
{
   public class EnterprisePositionDto
    {
        [DataMember]
        public Guid Id { get; set; }

        [DataMember]
        public String Name { get; set; }

        [DataMember]
        public EnterprisePositionDto Parent { get; set; }

        [DataMember]
        public Guid? ParentId { get; set; }

        [DataMember]
        public Guid OrganizationId { get; set; }

        [DataMember]
        public OrganizationDto Organization { get; set; }
        [DataMember]
        public int? SortingNumber { get; set; }
    }
}

File: Kms.Security.Portal/app/nodak/common/enterprisePosition/enterprisePosition.dto.ts
module nodak.common.models {
    export class EnterprisePositionDto {
        private id: any
        set Id(value)
        { this.id = value; }
        get Id()
        { return this.id; }

        private name: string
        set Name(value)
        { this.name = value; }
        get Name()
        { return this.name; }

        private parent: models.EnterprisePositionDto
        set Parent(value)
        { this.parent = value; }
        get Parent()
        { return this.parent; }

        private parentId: any
        set ParentId(value)
        { this.parentId = value; }
        get ParentId()
        { return this.parentId; }

        private organizationId: any
        set OrganizationId(value)
        { this.organizationId = value; }
        get OrganizationId()
        { return this.organizationId; }

        private organization: models.OrganizationDto
        set Organization(value)
        { this.organization = value; }
        get Organization()
        { return this.organization; }

        private sortingNumber;
        set SortingNumber(value) {
            this.sortingNumber = value;
        }
        get SortingNumber() {
            return this.sortingNumber;
        }

    }
}
